### ISSUE TITLE
clayui.com: Add more information to our Forms docs

### ISSUE THREAD

**Issue by pat270** [liferay/clay#5410]
From a private slack message:

> For example, I was creating a Client Extension with REACT , that has a form.
In order to build it with Clay, I found difficult to find in the documentation things like:
> - How to define a form and how to for example set the function that the form should trigger
> - No clear info about how to submit the form
> - No clear info about how to handle the onchange in the input fields
> - Not many examples of how to compose and style the form (there is just an HTML example)

> As an example, it would be great to have also the code using JS components of the examples in this page:
> [https://v2.clayui.com/docs/css-framework/satellites/forms/form-elements.htm](https://v2.clayui.com/docs/css-framework/satellites/forms/form-elements.html)

**Comment by matuzalemsteles** [comment]
I think we can use the same technical documentation that @marcoscv-work is working on, he's working on something like this.

**Comment by ethib137** [comment]
Yeah, the work Marcos is doing will tie in nicely with this. Additionally, it could be worthwhile for us to specifically consider the Objects apis when submitting a form, since that is likely the most common use case now for a customer creating a React form in Liferay inside a client extension.

**Comment by liferay-peds** [comment]
LPS Ticket: https://liferay.atlassian.net/browse/LPS-190182
